docs/schema/steel-mill-cert-test-report.json

```diff
@@ -0,0 +1,68 @@
+{
+  "@context": [
+    "https://w3id.org/did/v1",
+    "https://w3id.org/security/v2",
+    "https://w3c-ccg.github.io/vc-json-schemas/context/vc-json-schema-v0.0.jsonld"
+  ],
+  "type": "https://w3c-ccg.github.io/vc-json-schemas/schema/credential-schema.json",
+  "modelVersion": "1.0",
+  "id": "https://w3c-ccg.github.io/vc-json-schemas/schema/steel-mill-cert-test-report.json",
```

Comment:
should be a valid identifier according to https://w3c-ccg.github.io/vc-json-schemas/#id

I guess, given the vague language of the draft this is compliant...

Reply:
yes, i knew we would need to discuss this. IMO, we should not require this to be a DID URI, for many reasons, most of which are discussed on this thread:

https://github.com/hyperledger/aries-rfcs/issues/129

IMO, we should take this opportunity to document the discussion that happened in aries, and make it clear that the consumer has some choice in spec compliance here.

I will create an issue to track this.

Reply:
https://github.com/w3c-ccg/vc-json-schemas/issues/17

Reply:
Can I mark as resolved?

Reply:
yes